Detectors/FIT/common/simulation/src/Digitizer.cxx

```diff
@@ -18,26 +18,88 @@
 #include <algorithm>
 #include <cassert>
 #include <iostream>
+#include <optional>
 
 using namespace o2::fit;
 //using o2::fit::Geometry;
 
 ClassImp(Digitizer);
 
-double get_time(const std::vector<double>& times, double signal_width)
+double sinc(const double x)
```

Comment:
Check for exact equality of double is not reliable, better use some margin, e.g.
``return (std::abs(x)<1e-12) ? 1 : std::sin(x)/x;``

Also, it is better to inline such a short functions